chxtools/handlers2.py

```diff
@@ -96,8 +97,11 @@ def __call__(self, seq_id):
         # TODO Return a multi-dimensional PIMS seq.
         return EigerImages2(master_path, self._images_per_file, md=md)
 
-db.fs.deregister_handler('AD_EIGER')
-db.fs.deregister_handler('AD_EIGER2')
+db = Broker.named('chx')
+
+db.reg.deregister_handler('AD_EIGER')
+db.reg.deregister_handler('AD_EIGER2')
+
+db.reg.register_handler('AD_EIGER2', EigerHandler2)
```

Comment:
you could remove the previous two lines and add the `overwrite=True` flag to this line and bellow. Maybe we should get @danielballan 's blessing first ;-)

Reply:
Blessed!

Reply:
Done!